sound/soc/sof/pcm.c

```diff
@@ -574,6 +574,7 @@ static int sof_pcm_close(struct snd_soc_component *component,
 static int sof_pcm_new(struct snd_soc_component *component,
```

Comment:
can we add in the commit message a reminder of how memory allocation works if we use the parent to allocate the memory? I recall the ALSA core willl reclaim the memory when the card is closed, but how does it work if this is allocated by a different device?